netci.groovy

```diff
@@ -58,13 +58,20 @@ static void addWrappers(def myJob) {
 static void addArtifactArchiving(def myJob, String patternString, String excludeString) {
   myJob.with {
     publishers {
-      archiveArtifacts {
```

Comment:
I've since updated things based on the DSL here: https://jenkinsci.github.io/job-dsl-plugin/#path/job-publishers-flexiblePublish